packages/compass-import-export/test/setup.js

```diff
@@ -9,7 +9,9 @@ const chaiAsPromised = require('chai-as-promised');
 
```

Comment:
If you are moving to the new shared mocha config for tests in this package, you should not need this setup anymore as our shared configuration comes with a bunch of pre-configuration